pkg/controllers/management/feature/feature_handler.go

```diff
@@ -40,9 +42,26 @@ func (h *handler) sync(_ string, obj *v3.Feature) (*v3.Feature, error) {
 	if obj.Name == features.TokenHashing.Name() {
 		return obj, h.refreshTokens()
 	}
+
+	if obj.Name == features.Harvester.Name() {
+		return obj, h.toggleHarvesterNodeDriver(obj.Name)
+	}
 	return obj, nil
 }
 
+func (h *handler) toggleHarvesterNodeDriver(harvester string) error {
+	if val := features.GetFeatureByName(harvester).Enabled(); val {
```

Comment:
Should the node driver be disabled when the feature is turned off? In other words, I think it would be better to update the node driver whenever `Active` is different from the feature flag `Enabled` status.

Reply:
hey, @thedadams , the node driver can be kept enabled even the global Harvester feature is disabled. Please see the comment [here](https://github.com/rancher/rancher/issues/34886#issuecomment-931785510)